### ISSUE TITLE
Screen locking can be circumvented on Windows

### ISSUE THREAD

**Issue by boyvanduuren** [galadril/CaptureWolf#42]
The following sequence of actions will allow you to disable CaptureWolf without locking the screen:

1. Start CaptureWolf
2. Start Luring
3. Press ctrl + alt + delete, making sure to press it in that order and to pause shortly in between the presses
4. Keep pressed till you see the ctrl+alt+delete screen pop up
5. Use your mouse to select task manager
6. Select CaptureWolf
7. Click "End task"

Note that sometimes using the mouse after step 5 locks the screen. An alternative is to do everything from the keyboard as so:

5. Press down 4 times to select task manager
6. Press enter
7. Select CaptureWolf by pressing down
8. Press delete to end the task

**Comment by galadril** [comment]
Please try again on v1.0.18 💯

**Comment by galadril** [comment]
Please reopen or create new ticket if you find a new flaw ;) thanks
